src/conq/perception_lib/fetch_roboflow_data.py

```diff
@@ -13,6 +13,8 @@
 """
 
 from roboflow import Roboflow
+
+# Config stuff
 rf = Roboflow(api_key="22iMfgZpDlP0VFyWIc5t")
```

Comment:
Putting API keys in code and committing them is a very bad idea, it means literally anyone can access your roboflow data. It's not a huge deal for roboflow, but I recommend:
- re-generate your API key so that this one no longer works
- Store that in an environment variable (e.g in your bashrc put `export ROBOFLOW_API_KEY=asdfasdfasdfasdf`)
- In your code here use `os.environ['ROBOFLOW_API_KEY']`.

Reply:
Fixed